src/python/WMCore/ReqMgr/DataStructs/DefaultConfig/EDITABLE_SPLITTING_PARAM_CONFIG.py

```diff
@@ -5,7 +5,8 @@
              "LumiBased": {"lumis_per_job": '',
                           "halt_job_on_file_boundaries": True},
              "EventAwareLumiBased": {"events_per_job": '',
-                                     "max_events_per_lumi": '',
+                                     "max_events_per_lumi": '',  # has to be removed in ~HG1804
+                                     "job_time_limit": '',
```

Comment:
@ticoann this is something I'm not sure... requests created before this patch will have max_events_per_lumi, so we should still allow it to be changed, no?
I have to check what happens if we set max_events_per_lumi parameter for a new request (thus only job_time_limit set)(?) I guess nothing, but :-)

Reply:
Once agents are updated "max_events_per_lumi" will be ignored and if "job_time_limit" is set for old agent that will be ignored as well. The question is whether "max_events_per_lumi" is set always?

Reply:
It seems to be set in every single spec file, during the request creation.